akka-actor/src/main/scala/akka/pattern/BackoffSupervisor.scala

```diff
@@ -293,7 +295,7 @@ private[akka] trait HandleBackoff { this: Actor ⇒
   var restartCount = 0
 
   import BackoffSupervisor._
-  import context.dispatcher
+  private implicit val ec: ExecutionContext = context.dispatcher
```

Comment:
these two are only used to schedule on message receive, fine to keep original imports